pod/perlfunc.pod

```diff
@@ -3372,10 +3372,23 @@ X<glob> X<wildcard> X<filename, expansion> X<expand>
 In list context, returns a (possibly empty) list of filename expansions on
 the value of EXPR such as the standard Unix shell F</bin/csh> would do.  In
 scalar context, glob iterates through such filename expansions, returning
-undef when the list is exhausted.  This is the internal function
-implementing the C<< <*.c> >> operator, but you can use it directly.  If
-EXPR is omitted, L<C<$_>|perlvar/$_> is used.  The C<< <*.c> >> operator
-is discussed in more detail in L<perlop/"I/O Operators">.
+undef when the list is exhausted. If EXPR is omitted, L<C<$_>|perlvar/$_>
+is used.
+
+    # List context
+    my @txt_files  = glob("*.txt");
+    my @perl_files = glob("*.pl *.pm");
+
+    # Scalar context
+    while (my $file = glob("*.mp3")) {
+        # Do stuff
+    }
+
+Glob also supports an altnernate syntax using C<< < >> C<< > >> as
+delimiters. While this syntax is supported, it is recommended that you
+use C<glob()> instead as it is more readable and searchable.
```

Comment:
Functions should not be followed by empty parentheses in sentences according to the new style guide: https://perldoc.perl.org/perldocstyle#Function-names